cmd/humioctl/filter_alerts_install.go

```diff
@@ -0,0 +1,86 @@
+// Copyright © 2020 Humio Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/humio/cli/api"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+func newFilterAlertsInstallCmd() *cobra.Command {
+	var (
+		filePath, url, name string
+	)
+
+	cmd := cobra.Command{
+		Use:   "install [flags] <view>",
+		Short: "Installs a filter alert in a view",
+		Long: `Install a filter alert from a URL or from a local file.
+
+The install command allows you to install filter alerts from a URL or from a local file, e.g.
+
+  $ humioctl filter-alerts install viewName --name filterAlertName --url=https://example.com/acme/filter-alert.yaml
```

Comment:
Why do we allow overriding the name?

Reply:
I can't see the need for it, unless you want to create multiple alerts with the same "template". Will remove it for now.